functions/src/models/organizations/teams.ts

```diff
@@ -0,0 +1,59 @@
+import { FirestoreDataConverter } from '@google-cloud/firestore';
+import { IsArray, IsNotEmpty, IsString } from 'class-validator';
+import { firestore } from 'firebase-admin';
+
+import { OrganizationType } from './index';
+import DocumentData = firestore.DocumentData;
+
+export interface ITeam extends DocumentData {
+  name: string;
+  types: OrganizationType[];
```

Comment:
I don't think we want to work on this part of the model yet, it takes focus away from the MVP, and we can add it later. I think we also want to think about this a little more. E.g. what's the difference between a Team and Organization, can't they just be the same thing? etc...

Let's remove this for now.

Reply:
This is just a placeholder so that we can leave some mental room for orgs (and teams inside orgs) when thinking about users and other models that are used in the MVP.